### DISCUSSION TITLE
Option to configure the API rules for the Collections operations

### DISCUSSION THREAD

**Discussion by mahirchavda** [pocketbase/pocketbase#2041] [Q&A] [answered]
Currently, Only admins can access following actions: List Collections, CRUD Collection, Import Collections. 
It would be great if we have API rules configuration for above actions as well; Similar to API Rules.

Use Case: I'm trying to build my own GUI for non-admin users and for that I need List collections capability for particular user roles

**Comment by ganigeorgiev** [comment, marked as answer]
I don't understand your use case.

Collections or any other db schema changes are not intended to be performed by non-admin users.

If you need "roles" for the admins, this eventually may be added in the future (together with API keys/fixed tokens) since it was requested several times already, but for now remains out of the scope for v1.

**Reply by mahirchavda** [reply]
I am mainly interested in the List Collections. I want to list down all the collections and it's schema.

**Reply by ganigeorgiev** [reply]
If you need only a read-only access to the collections configuration, you can create a "View collection" for the internal `_collections` table:

```sql
SELECT id, name, schema FROM _collections
```
